### PR TITLE
UIBULKED-115: In-app bulk edit form - holdings location

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
## Purpose

[UIBULKED-115](https://issues.folio.org/browse/UIBULKED-115)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,11 +10,13 @@
 * [UIBULKED-124](https://issues.folio.org/browse/UIBULKED-124) User-records - confirm changes - in app.
 * [UIBULKED-110](https://issues.folio.org/browse/UIBULKED-110) Populate holdings record identifiers dropdown
 * [UIBULKED-129](https://issues.folio.org/browse/UIBULKED-129) Preserve column selection from preview to Are you sure? form and confirmation page
+* [UIBULKED-115](https://issues.folio.org/browse/UIBULKED-115) In-app bulk edit form - holdings location.
 
 ## [2.0.3](https://github.com/folio-org/ui-bulk-edit/tree/v2.0.2) (2022-09-09)
 * [UIBULKED-144](https://issues.folio.org/browse/UIBULKED-144) Update behavior of uploading CSV file with updated records
 * [UIBULKED-141](https://issues.folio.org/browse/UIBULKED-141) Holdings record - preview of matched records
 
+
```
Comment (uladislausamets) [inline CHANGELOG.md:19]: 👀

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js [hunk 2]
```diff
@@ -56,6 +57,14 @@ export const BulkEditInApp = (
             getFilteredFields={getFilteredFields}
           />
         )}
+
+        {typeOfBulk === CAPABILITIES.HOLDINGS && (
+          <HoldingsForm
+            onContentUpdatesChanged={onContentUpdatesChanged}
+            typeOfBulk={typeOfBulk}
```
Comment (uladislausamets) [inline src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:64]: Is `typeOfBulk` used in `HoldingsForm` ?

File: CHANGELOG.md (hunk 1 above)
Comment (UladzislauKutarkin) [reply CHANGELOG.md:19]: @uladislausamets Thanks for noticing that, I missed it when was busy resolving the conflicts.

File: src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js:64]: Ye, It's an unused prop. I will remove it

File: src/components/BulkEditActionMenu/BulkEditActionMenu.js [hunk 3]
```diff
@@ -119,6 +119,16 @@ const BulkEditActionMenu = ({
             </Icon>
           </Button>
         )}
+        {capabilities === CAPABILITIES.HOLDINGS && (
+        <Button
+          buttonStyle="dropdownItem"
+          onClick={buildButtonClickHandler(onUserEdit)}
```
Comment (uladislausamets) [inline src/components/BulkEditActionMenu/BulkEditActionMenu.js:125]: `onUSerEdit` means changing users, but here we open form for editing holdings

**Review by vashjs — APPROVED** [review]

**Review by uladislausamets — APPROVED** [review]

**Pull request merged**